concent_api/core/payments/bankster.py

```diff
@@ -474,3 +477,18 @@ def discard_claim(deposit_claim: DepositClaim) -> bool:
         subtask_id=deposit_claim.subtask_id,
     )
     return claim_removed
+
+
+def validate_list_of_transaction_timestamp(
```

Comment:
one more thing - please move it to `validation` module

Reply:
ehh, okey. I will need to change commit name then. So `push --force` here we go! xD

Reply:
Done